app/workers/pair_requests/auto_expire_worker.rb

```diff
@@ -1,4 +1,4 @@
-class PairRequest
+module PairRequests
```

Comment:
We have to take extra precautions when editing a worker's name. This is because when a worker is enqueued, the worker's name is one of the things that's saved into queue. When the worker is dequeued and processed, its name is used to look up the code that should be run.

This causes an issue though if we change the name while workers are in the queue. To safely rename, we have to temporarily alias the class until we're sure no workers with the old  names are present in the queue.

Can [see here](https://github.com/sidekiq/sidekiq/wiki/FAQ#how-do-i-safely-rename-a-worker) in the sidekiq wiki for how this is done. You should do this for each of the changing worker names. Then we can come back later with a new PR in ~1 week that cleans up the old names completely.

Reply:
Will do

Reply:
I’ve set the aliases and a reminder for myself to remove them after a week.
Would I be able to check the queue myself?

Reply:
I can't even view it unfortunately :(

The production queue is only viewable for admin users.